ResearchKit/ActiveTasks/HKSample+ORKJSONDictionary.m

```diff
@@ -142,6 +142,7 @@ @implementation HKCorrelation (ORKJSONDictionary)
 
 - (NSDictionary *)ork_JSONDictionaryWithOptions:(ORKSampleJSONOptions)options sampleTypes:(NSArray *)sampleTypes units:(NSArray *)units {
     NSMutableDictionary *mdict = (NSMutableDictionary *)[self ork_JSONDictionaryWithOptions:options unit:nil];
+    NSAssert([mdict isKindOfClass:[NSMutableDictionary class]], @"");
```

Comment:
I'd rather this assert need stayed in, *or* if we changed the base class to provide a differently named implementation that explicitly returns a mutable dictionary.